### PR TITLE
Improved the status output of the `atmosphere` service script.

### PR DESCRIPTION
**Pull request by lenards**
The script now reports the status of other (included/dependent) daemons in the same order that is used for `start|stop|restart`. 

Also, an indication between the uWSGI processes for atmosphere & troposphere is included (along with clarifying output for nginx, celery, redis, etc). 

In addition to that, a `status` method for `celerybeat` is now included from the generic init script example within the celery [repo](https://github.com/celery/celery/blob/master/extra/generic-init.d/celerybeat).

Tested and verified locally on vagrant. 

Targeting `master` (aka Larping Loon) release.

### CODE REVIEW

**Comment by lenards** [comment]
Related back to [ATMO-1043](https://pods.iplantcollaborative.org/jira/browse/ATMO-1043).

File: api/v2/views/machine_request.py [hunk 1]
```diff
@@ -34,9 +67,9 @@ def perform_create(self, serializer):
                 status=status,
                 old_status="processing",
```
Comment (steve-gregory) [inline api/v2/views/machine_request.py]: This line considered harmful (?)

**Comment by steve-gregory** [comment]
uWSGI part LG2M. Not so sure about machine request.

File: api/v2/views/machine_request.py (hunk 1 above)
Comment (lenards) [inline api/v2/views/machine_request.py]: Yes, still harmful. I'm not introducing that line here.

Comment (steve-gregory) [inline api/v2/views/machine_request.py]: Ok LG2M!

**Comment by steve-gregory** [comment]
:shipit:

**Comment by lenards** [comment]
Thanks.

**Pull request merged**